### PR TITLE
run the elasticsearch tests

### PR DESCRIPTION
**Pull request by czue**
and some other small tweaks

### CODE REVIEW

File: localsettings.example.py [hunk 1]
```diff
@@ -205,7 +205,9 @@
 #    'couchdebugpanel', # Adds couch info to said toolbar
 #    'devserver',       # Adds improved dev server that also prints SQL on the console (for AJAX, etc, when you cannot use debug_toolbar)
 #    'django_cpserver', # Another choice for a replacement server
-#    'dimagi.utils'     
+#    'dimagi.utils'
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

File: localsettings.example.py [hunk 2]
```diff
@@ -205,7 +205,9 @@
...
+#     'testapps.test_elasticsearch',
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

File: localsettings.example.py [hunk 3]
```diff
@@ -205,7 +205,9 @@
...
+#     'testapps.test_pillowtop',
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

File: localsettings.example.py [hunk 4]
```diff
@@ -205,7 +205,9 @@
 #    'couchdebugpanel', # Adds couch info to said toolbar
 #    'devserver',       # Adds improved dev server that also prints SQL on the console (for AJAX, etc, when you cannot use debug_toolbar)
 #    'django_cpserver', # Another choice for a replacement server
-#    'dimagi.utils'     
+#    'dimagi.utils',
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

File: localsettings.example.py [hunk 5]
```diff
@@ -205,7 +205,9 @@
...
+#    'testapps.test_elasticsearch',
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

File: localsettings.example.py [hunk 6]
```diff
@@ -205,7 +205,9 @@
...
+#    'testapps.test_pillowtop',
```
Comment (dimagimon) [inline localsettings.example.py]: E122 continuation line missing indentation or outdented

**Comment by czue** [comment]
:white_check_mark:

**Comment by czue** [comment]
@benrudolph

**Comment by benrudolph** [comment]
nice

**Pull request merged**